cortico.js

```diff
@@ -1230,4 +1232,55 @@ function plusSignFromCache() {
   }
 }
 
+async function init_recall_button() {
+  const statusOption = document.querySelector("select[name='status']");
+  var statusValue = statusOption.options[statusOption.selectedIndex].text;
+  console.log("echo", statusValue)
```

Comment:
maybe, console.log("status:", statusValue) is more helpful if someone else sees this?